code/graphics/gropengltnl.cpp

```diff
@@ -2025,13 +2025,6 @@ void opengl_tnl_set_material(int flags, uint shader_flags, int tmap_type)
 			vm_vec_unrotate(&pos, &G3_user_clip_point, &Eye_matrix);
 			vm_vec_add2(&pos, &Eye_position);
 
-			vec4 clip_plane_equation;
-
-			clip_plane_equation.a1d[0] = normal.a1d[0];
-			clip_plane_equation.a1d[1] = normal.a1d[1];
-			clip_plane_equation.a1d[2] = normal.a1d[2];
-			clip_plane_equation.a1d[3] = vm_vec_mag(&pos);
-
```

Comment:
The shader receives all necessary information for the clip plane via the `clip_normal` and `clip_position` uniforms. This variable contains the HNF form of the plane which isn't needed by any code.